qutebrowser/browser/downloads.py

```diff
@@ -727,6 +736,23 @@ def _after_create_parent_question(self,
         else:
             self._after_set_filename()
 
+    def _get_conflicting_downloads(self):
+        """Returns a list of other active downloads with the same name."""
+        return [
+            download for download in
+            objreg.get('download-model', scope='window',
+                       window='current')._all_downloads()
```

Comment:
You don't need the `._all_downloads()` here because DownloadModel is an iterator. (And it'll make a lint error go away.)